src/javascript/app_2/App/Components/Elements/Calendar/panels/calendar_days.jsx

```diff
@@ -5,10 +5,10 @@ import CalendarPanelTypes from './types';
 import { localize }       from '../../../../../../_common/localize';
 import { padLeft }        from '../../../../../../_common/string_util';
 
-const getDays = ({ calendar_date, date_format, max_date, min_date, onClick, selected_date }) => {
+const getDays = ({ calendar_date, date_format, max_date, min_date, start_date, onClick, selected_date }) => {
     const dates = [];
     const days  = [];
-    const moment_today       = moment().utc();
+    const moment_today       = moment().utc().set({ hour: 0, minute: 0, second: 0, millisecond: 0 });
```

Comment:
@negarn 
Also, you can use `moment().utc().startOf('day')` to set time to 12:00 a.m